### ISSUE TITLE
Pods not installing correctly with RN autolinking

### ISSUE THREAD

**Issue by awatt-cox** [useriq-com/useriq-react-native#7]
When installing the library on React Native >= 0.60, the "pod install" command does not install the latest version of UserIQ or UseriqReactNative. It is not consistent behavior. On our development team, we had these results:

**Step 1:**
on a new Git branch, the first developer ran:
npm install @useriq/useriq-react-native --save
pod install

Result: The latest pods were installed. Everything worked correctly.

**Step 2:**
2nd developer checked out the branch for pull request review. 2nd Developer ran:
npm i
pod install

Result: iOS project would not build with error:
error: no visible @interface for 'UserIQSDK' declares the selector 'logout'
    [[UserIQSDK sharedInstance] initWithAPIKey];
     ~~~~~~~~~~~~~~~~~~~~~~~~~~ ^~~~~~
1 error generated.

After some research, it was discovered that UserIQ v2.2.0 was installed.

running "pod update" fixed the issue.

STEP 3:
3rd developer checked out the branch and ran:
npm i
pod install

RESULT: iOS project would not build. Similar error as above. On this developer machine, UserIQ v2.0.1 was installed. Again, "pod update" fixed the issue.

CONCLUSIONS:
The autolinked pods are not installing the newest version of UserIQ. The version that is downloaded on a fresh pod install seems to be random. Only running "pod update" fixes the issue.

ADDITIONAL NOTE:
- When a new version of the library is posted, it breaks the build again. v2.6.0 was released this afternoon, which again broke our iOS builds. It seems that UseriqReactNative updates correctly, but UserIQ does not, which breaks the build. This will be problematic for our CI pipeline.

**Comment by aravindgs** [comment]
This issue is currently being tracked in [Zendesk #4444](https://useriq.zendesk.com/agent/tickets/4444). Changes are already in the development branch. Will update once release is done

**Comment by aravindgs** [comment]
Issue resolved in #9. Please update UseriqReactNative library to 2.6.2
